src/components/Error/Error.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './Error.css' 
+import { Link } from 'react-router-dom' 
+
+const Error = () => {
+  return (
+    <div className='error-container'>
+      <p className='error-message'>404: Page Not Found</p>
+      <Link to='/'><button className="go-back-to-cities">GO BACK TO CITIES</button></Link>
+    </div>
+  )
+}
+
```

Comment:
I think if we pass the error status code and the status text as props we can dynamically add them to this component.